diff --git a/language/go/generate.go b/language/go/generate.go
index 2892948..b5407ee 100644
--- a/language/go/generate.go
+++ b/language/go/generate.go
@@ -545 +545 @@ func (g *generator) maybeGenerateToolLib(lib *rule.Rule, pkg *goPackage) *rule.R
-	toolLib := rule.NewRule("go_tool_library", "go_tool_library")
+	toolLib := rule.NewRule("go_default_library", "go_default_library")
diff --git a/language/go/resolve.go b/language/go/resolve.go
index 50bfb0a..b6318b1 100644
--- a/language/go/resolve.go
+++ b/language/go/resolve.go
@@ -22 +21,0 @@ import (
-	"log"
@@ -82 +81 @@ func (gl *goLang) Resolve(c *config.Config, ix *resolve.RuleIndex, rc *repo.Remo
-	deps, errs := imports.Map(func(imp string) (string, error) {
+	deps, _ := imports.Map(func(imp string) (string, error) {
@@ -97,3 +95,0 @@ func (gl *goLang) Resolve(c *config.Config, ix *resolve.RuleIndex, rc *repo.Remo
-	for _, err := range errs {
-		log.Print(err)
-	}
